### ISSUE TITLE
Episode page - hosts/guests layout change

### ISSUE THREAD

**Issue by gerbrent** [JupiterBroadcasting/jupiterbroadcasting.com#226]
JB has suggested it would be really great to change the layout of the hosts on Episode pages. We should also consider guests in this scenario..

#### Currently
Hosts (and guests) currently occupy a large vusal weighting in the first "page" of episode pages, pushing Episode Links far down the page. 

#### Preferred
It would be great to see Hosts take less visual space to allow Episode Links to rise nearer to the top of the page. Episode Links are high priority content...

#### Suggestion
Host profiles can be made smaller, and moved to the right sidebar - below Sponsors, above Tags
Smaller Host and Guest profile cards: ...perhaps near the vertical dimensions of sponsors currently.

#### Guests - *where oh shalt thy go?*
Guests are a little more special. Hosts don't change often, so demoting their priority is quite acceptable. Guests, however, feel important indeed. What then to do?

I would love to see them in the right frame aswell, to gauge how that feels on a show like LUP w 3 hosts, and sometimes multiple guests.
Open to alternatives.

---

## [LUP 466](https://jupiterbroadcasting.net/show/linux-unplugged/466/) - 3 hosts, 1 guest

![image](https://user-images.githubusercontent.com/7018928/183118665-7b1cfd1d-c191-4884-b87e-e77d1099cb4c.png)

![image](https://user-images.githubusercontent.com/7018928/183119377-c343f68c-9fa4-4a86-99f9-572bd5d5bb5b.png)

## [LUP 434](https://jupiterbroadcasting.net/show/linux-unplugged/434/) - 3 hosts, 3 guests

![image](https://user-images.githubusercontent.com/7018928/183119120-3dca984c-1e07-4205-b094-983776c6a53d.png)

![image](https://user-images.githubusercontent.com/7018928/183119254-a60be409-c37b-4ed4-8edc-495cc1b2358b.png)

## [Extras 87](https://jupiterbroadcasting.net/show/jupiter-extras/87/) - 1 host, 1 guest

![image](https://user-images.githubusercontent.com/7018928/183118839-a5a82507-56f5-417f-9c60-5ef0f01c35df.png)

![image](https://user-images.githubusercontent.com/7018928/183119440-aab97e66-36ea-4a13-bb4e-1dae9917acd4.png)

**Comment by CGBassPlayer** [comment]
So I know this was moved to a 2.0 milestone, but I took a look at it anyway. This is what I have so far...

![hosts and guests on 1 line](https://user-images.githubusercontent.com/34293941/183510799-65dcd055-7a9c-4e8e-ae91-2dc1c14ee11e.png)

Using the `smallest` card for the host and creating an identical card for the guests. That way everything could be pushed everything to one line. I still need to fix up the columns to make it look better on smaller screens. My plan is to move the Guest *below* the hosts if everyone does not fit on 1 line.

![hosts and guest wrapping oddly](https://user-images.githubusercontent.com/34293941/183510882-e97b6bbc-2fdd-4918-b3a1-b937c4dbd631.png)

It also made me notice how the tags and links for the episode are tied together vertically. Little more work required, but something that shouldn't be too bad to tackle either. Just playing with some columns.

**Comment by gerbrent** [comment]
Nice work so far!

I have a big question:

* Currently, we have two types of profile layouts:
    * circular profiles - Show pages, 
    * card-based profiles - Episode pages, Host/Guest pages

I see here you've replaced the Episode card profiles w circular ones, which mimic the [Fireside](https://coder.show/) style. 

For circular profiles:
Advantage: less physical space, more compact
Disadvantage: umm.........?

I am in support of this approach, with cards remaining for Host/Guest pages, and circular profiles for Episode/Show pages.
Anyone else have a strong opinion either way?

In conclusion: I really like this change! 🎯

**Comment by CGBassPlayer** [comment]
@gerbrent I have a question while I'm working on this...

![Screenshot of updated layout on desktop](https://user-images.githubusercontent.com/34293941/184145990-994430ba-da3e-4f2f-8600-8b289124e6a0.png)

I shrunk the hosts and guests, but I also made it so the episode links are no longer starting at the same height as the tags, and are instead starting under the hosts (working out bugs on mobile view). 

BUT when it goes to mobile, the sponsor now show *below* the episode links. 

![Screenshot of updated layout on mobile](https://user-images.githubusercontent.com/34293941/184147304-6c944bbc-b6b3-43e5-b09f-ea1962df01a6.png)

My gut tells me JB would want the sponsor to be above the episode links, but I figured I would confirm that before I go through the work to move it up on mobile.

**Comment by ChrisLAS** [comment]
> My gut tells me JB would want the sponsor to be above the episode links, but I figured I would confirm that before I go through the work to move it up on mobile.

Yeah, that's a good point. We want it to be easy to find and support our shows via sponsors.  So they should be above the links in that layout above.

**Comment by ChrisLAS** [comment]
> I am in support of this approach, with cards remaining for Host/Guest pages, and circular profiles for Episode and Show pages. Anyone else have a strong opinion either way?
> 
> In conclusion: I really like this change! 🎯

I'd be inclined to go with any approach that gets the hosts and guests' profile pics taking up the least amount of space possible.

**Comment by gerbrent** [comment]
thanks for all the work and for asking so clearly!

Above is best.....
